docs/concepts.rst

```diff
@@ -32,3 +32,53 @@ Variable
 A *variable* represents a process measure of one :ref:`appliance <appliance>` of which
 timeseries data is captured and stored in the ENLYZE platform. One appliance may have
 many variables, whereas one variable is only associated with one appliance.
+
+
+.. _production_order:
+
+Production Order
+----------------
+
+A *production order* represents the goal of producing a certain quantity of a
+given :ref:`product <product>`. Production orders are usually created in an ERP
+system or MES of the customer and then synchronized into the ENLYZE platform.
+They are referenced by an identifier which oftentimes is a short combination of
+numbers and/or characters, like FA23000123 or 332554.
+
+In the ENLYZE platform, a production order always encompasses the production of
+one single :ref:`product <product>` on one single :ref:`appliance <appliance>`.
+
+
+.. _production_run:
+
+Production Run
+--------------
+
+A *production run* represents the real allocation of a :ref:`production order
```

Comment:
While technically correct, I'm not sure how easy it is to grasp this.

Reply:
Maybe something along the lines of:

A production run represents an time frame during which an appliance was working towards completing production order[..]?

Reply:
Hm, maybe like this?

> A production run represents **a time span when an appliance was allocated to a production order.**
> ....
> A production order oftentimes **is not finished in one go** due to interruptions like a breakdown or a weekend. Each of these individual, continous allocations is called a production run in the ENLYZE platform. **Thus, there can be multiple production runs for one production order.**

Bold sections are my proposed changes.

Reply:
I'll create a new multi-line suggestion